src/WarpedPsf.cc

```diff
@@ -89,11 +89,11 @@ geom::Box2I computeBBoxFromTransform(geom::Box2I const bbox, geom::AffineTransfo
     }
 
     // We want to guarantee that the output bbox has odd dimensions, so instead
-    // of using the Box2I converting constructor directly, we round the center
-    // point of the floating point box and dilate by its half-dimensions.
+    // of using the Box2I converting constructor directly, we start with (0, 0)
+    // and dilate by the floating-point box's half-dimensions.
     geom::Extent2I out_half_dims = geom::floor(0.5*out_box_fp.getDimensions());
     geom::Box2I out_box;
-    geom::Point2I out_center(out_box_fp.getCenter());
+    geom::Point2I out_center(0, 0);
     out_box.include(out_center);
```

Comment:
Any reason that you decided to define `out_box` this way instead of using `out_box = Box2I(Point2I(0,0), Extent2I(1,1))`?

Reply:
Not really; it just fit my mental model of what I was doing better.